quic/s2n-quic-crypto/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "s2n-quic-crypto"
-version = "0.21.0"
+version = "0.21.1"
```

Comment:
Shouldn't this be a minor version bump as well? Basing this off of "For example if s2n-quic-core has a minor version bump, all the crates at the same level as s2n-quic-core will also have a minor version bump."

Reply:
Nope, that was a deliberate decision. As an extreme example, consider what happens when one of our dependencies has a major (breaking) change. In that situation we would not bump the major version of s2n-quic, because there was no breaking change for s2n-quic.

Likewise, I'd argue that a feature bump for a dependency crate does not necessarily imply a feature bump for the consuming crate. 

The way that the feature bumps are actually calculated, is that a crate will have a feature (minor) bump if some commit with `feat` in the title touched the crate.

Reply:
I guess if there is a reason for that policy then I can go ahead and change the script to reflect that, but I do worry the customers are losing out on a bit of information. E.g. with that policy `s2n-quic` will always have a feature bump as long as _any_ crate has had a feature bump.

Reply:
Yeah I haven't made up my mind if this is what we want or not... On the one hand, if we treat internal crates as just any other arbitrary crate then it makes sense to just do a patch bump. _However_, customers don't consume our internal crates directly, they consume the `s2n-quic` crate. Our internal crates mostly exist for organizational purposes and flexibility (like if we wanted to be in the kernel we could use the `s2n-quic-core` crate without having to split apart the codebase or expose all of those unstable APIs in the main crate). So in a world where we didn't split the crates up, if we implemented a `feat` at all in the codebase (which would be a single crate), it would warrant a `minor` bump.

Reply:
I see what you're saying, but right now it kind of feels like we have the worst of both worlds, because we have to calculate individual updates for each crate but they still all have different versions.

If we want to treat everything as a single codebase, then I think we should just have version numbers match across all of the crates, like we do with the rust bindings in s2n-tls

Reply:
I'd be fine with all of the minor/patch version numbers matching. We just don't want to have any of the internal crates `>=1.x.x` at the moment

Reply:
For the team being, I'll just go with the current method and add an issue for deciding how we want to do this.